### PR TITLE
Porting remaining MRTK2 constraints for completeness

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
We've gotten requests to add the missing MRTK2 constraints into the spatial manipulation package. 

This PR adds those scripts as is, making some additional changes to some other files to smooth out the transition.

These constraints are **not intended to be maintained long term**, but are added for now to allow MRTK3 users to use the old MRTK2 constraints while we refactor the existing system.

## Changes
- Fixes: #10779 #10762

### CODE REVIEW

File: com.microsoft.mrtk.spatialmanipulation/Constraints/ConstraintManager.cs [hunk 1]
```diff
@@ -10,6 +10,7 @@ namespace Microsoft.MixedReality.Toolkit.SpatialManipulation
     /// <summary>
     /// Manages constraints for a given object and ensures that Scale/Rotation/Translation 
     /// constraints are executed separately.
+    /// We're looking to rework this system in the future. These existing components will be deprecated in then.
```
Comment (keveleigh) [inline com.microsoft.mrtk.spatialmanipulation/Constraints/ConstraintManager.cs:13]: The last sentence (throughout these components) might have an extra word?

```suggestion
    /// We're looking to rework this system in the future. These existing components will be deprecated then. 
```

File: com.microsoft.mrtk.spatialmanipulation/Constraints/RotationAxisConstraint.cs.meta [hunk 2]
```diff
@@ -0,0 +1,11 @@
+fileFormatVersion: 2
+guid: 4461cef9324cf904299f3d04a3861659
```
Comment (keveleigh) [inline com.microsoft.mrtk.spatialmanipulation/Constraints/RotationAxisConstraint.cs.meta:2]: Just to double check, were these regenerated from MRTK2?

Comment (RogPodge) [reply com.microsoft.mrtk.spatialmanipulation/Constraints/RotationAxisConstraint.cs.meta:2]: Yeah they are regenerated, should they not be?

Comment (keveleigh) [reply com.microsoft.mrtk.spatialmanipulation/Constraints/RotationAxisConstraint.cs.meta:2]: Regeneration is good! Thanks!

**Review by keveleigh — APPROVED** [review]

**Pull request merged**